### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by PavelBal**

### CODE REVIEW

File: vignettes/standard-workflow.Rmd [hunk 1]
```diff
@@ -24,7 +24,7 @@ knitr::opts_knit$set(
 library(esqlabsR)
 ```
 
-### Defining scenarios
```
Comment (PavelBal) [inline vignettes/standard-workflow.Rmd:27]: @Felixmil just tagging you along, you can read the docs if you want to. The other changes are coming from the other PR, gut mixed up a bit.

File: vignettes/standard-workflow.Rmd [hunk 2]
```diff
@@ -33,6 +33,66 @@ application protocol, and (optionally) the physiology of the simulated
 individual or population. To simplify scenarios set up, all these
 information are stored in excel files with defined structure.
 
+The step wise approach of setting up a new simulation scenario is shown
```
Comment (PavelBal) [inline vignettes/standard-workflow.Rmd:36]: @VanessaBa  From here on.

**Comment by Felixmil** [comment]
@PavelBal You may want to merge this branch into the other one ?

**Comment by PavelBal** [comment]
@Felixmil Vice versa - I want the docu being reviewed separately from the functional code changes.

**Review by Felixmil — APPROVED** [review]

**Pull request merged**